### DISCUSSION TITLE
Can we change the database type after installtion?

### DISCUSSION THREAD

**Discussion by Suleman-Elahi** [nocodb/nocodb#6367] [Q&A] [answered]
I installed using the default Docker command that uses the SQLite database.

I have some work in that instance. Now, I want to change the database to MySQL.... is it possible?

Or, do you think SQLite is good enough for a team of 5-10 people ?

**Comment by wingkwong** [comment, marked as answer]
> I have some work in that instance. Now, I want to change the database to MySQL.... is it possible?

You can change to MySQL by specifying `NC_DB`. However, you need to migrate the schema and the meta data yourself (if you need to preserve the meta data)

**Reply by Suleman-Elahi** [reply]
Can you please tell where to look for SQLite file in the Docker image? Or is it stored locally on the server somewhere.

**Reply by wingkwong** [reply]
If you haven't specified `NC_DB`, by default a sqlite file will be created under `/usr/app/data`.

**Reply by Suleman-Elahi** [reply]
Okay thanks.. I will use this database converter .. hope it works: https://www.rebasedata.com/convert-sqlite-to-mysql-online
